include/net_eth.h

```diff
@@ -53,6 +53,7 @@ namespace type
 {
 
 static constexpr unsigned short int Aoe = 0x88a2;
+static constexpr unsigned short int Aoe = 0x0806;
```

Comment:
Tu nie powinno byc Arp ? bo narazie jest 2x Aoe i to jeszcze z innym hex'em.

Reply:
masz racje, poprawiam